app/src/test/kotlin/com/wire/android/util/UriUtilTest.kt

```diff
@@ -38,6 +38,23 @@ class UriUtilTest {
         assert(expected == actual)
     }
 
+    @Test
+    fun givenLink_whenTheLinkStartsWithMailTo_thenReturnsTheSameLink() {
+        val input = "mailto:alice@wire.com"
+        val expected = "mailto:alice@wire.com"
+        val actual = normalizeLink(input)
+        assert(expected == actual)
+    }
+
+    @Test
+    fun givenLink_whenTheLinkIsWireDeepLink_thenReturnsTheSameLink() {
+        val input = "wire://access/?config=https://nginz-https.elna.wire.link/deeplink.json"
+        val expected = "wire://access/?config=https://nginz-https.elna.wire.link/deeplink.json"
+        val actual = normalizeLink(input)
+        assert(expected == actual)
+    }
+
+
     @Test
     fun givenLink_whenTheLinkStartsWithRandomSchema_thenReturnsTheSameLink() {
         val randomString = Random.string(Random.nextInt(5))
```

Comment:
issue: This is flaky.

`Random.netInt(5)` can return 0.

Which will lead to `normalizeLink` adding `https://` to the beginning of the URI:
```
Expected :://google.com
Actual   :https://://google.com
```

In general: using randomness in tests is bad. Please avoid doing so.
If we have defined scenarios, we can write tests for all of them, instead of leaving flakyness to show up in the future.